app/accounts/models.py

```diff
@@ -11,7 +11,7 @@ class UserProfile(models.Model):
     Example of extra information can be google user id, facebook user id, etc.
     """
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, unique=True, on_delete=models.CASCADE)
     custom_auth_id = models.TextField(blank=True)
     facebook_oauth_id = models.TextField(blank=True)
     google_oauth_id = models.TextField(blank=True)
```

Comment:
@andela-ooshodi That can be changed sometime in the future. For now, I think we should go with this since we do not know exactly the maximum length of those id strings coming in.